src/main/webapp/css/projects.css

```diff
@@ -0,0 +1,9 @@
+.project-preview {
+  transition: .05s;
+}
+
+.project-preview:hover {
+  box-shadow: 0px 0px 5px 1px rgba(232,232,232,1);
+  -moz-box-shadow: 2px 2px 5px 1px rgba(232,232,232,1);
```

Comment:
Should we consider removing the box-shadow prefixes?: https://css-tricks.com/do-we-need-box-shadow-prefixes/
(seems like less than 0.3% of Firefox and Safari users are still on versions that require the prefix)

Reply:
Also super super nit but the ordering of declarations as described on the style sheet seems to show that vendor-specific prefixed versions of a property should come before the non-prefixed property https://google.github.io/styleguide/htmlcssguide.html

Reply:
Yeah I think you're right about that. So I think then the rule is vendor-specific come before but alphabetized. Nice!

Reply:
And that first article you sent is great. I agree it seems like they're unnecessary. Will remove.